### DISCUSSION TITLE
Internal Server Error after I install and initialize Next.js app.

### DISCUSSION THREAD

**Discussion by davidberka** [vercel/next.js#23890] [Help]
Hi, everytime I install and initialize a Next.js project I get an Internal Server Error. Does anybody know what it could be?
Thanks.
![err](https://user-images.githubusercontent.com/77415433/114305564-ec1eff80-9ad8-11eb-9981-0ca24f0fde26.png)

**Comment by apollonian** [comment]
Could you add a reproducible example? That would be very helpful. 
Do include the development server logs as well! 😇

**Reply by davidberka** [reply]
I don't modify the code at all. I just do npx create-next-app my-app, cd my-app, npm run dev and that's it. I have the latest version of node installed (14.16.1), then I have:
next: 10.1.3,
react: 17.0.2,
react-dom: 17.0.2
![err](https://user-images.githubusercontent.com/77415433/114319064-21930f80-9b10-11eb-9603-0057f2f7f7aa.png)
![err2](https://user-images.githubusercontent.com/77415433/114319065-222ba600-9b10-11eb-8ae3-7e93150e44c2.png)

**Reply by apollonian** [reply]
This issue might be relevant: https://github.com/vercel/next.js/issues/16535#issuecomment-786007339

**Reply by davidberka** [reply]
Because everytime I go into the documents directory the error pops up.
